async/kafka/group/group.go

```diff
@@ -32,12 +33,12 @@ func New(name, group string, topics, brokers []string, oo ...kafka.OptionFunc) (
 		return nil, errors.New("group is required")
 	}
 
-	if len(brokers) == 0 {
-		return nil, errors.New("provide at least one broker")
+	if len(brokers) == 0 || containsEmtpyValue(brokers) {
```

Comment:
Would it make sense to create a function that does both?

```suggestion
	if isEmpty(brokers) {
```

Reply:
Yes, it definitely makes sense, especially if we move this logic under a common package.

Probably from the naming perspective, we may be more specific `IsStringSliceEmpty` to leave a naming space for the future validators.

WDYT?

Reply:
sounds ok!